README.md

```diff
@@ -125,7 +126,8 @@ Netdata has a quite different approach to monitoring.
 Netdata is a monitoring agent you install on all your systems. It is:
 
 -   a **metrics collector** - for system and application metrics (including web servers, databases, containers, etc)
--   a **time-series database** - all stored in memory (does not touch the disks while it runs)
+-   a **database engine** - recent metrics stored in memory while historical metrics "spilled" to disk for 
```

Comment:
either "spilled" to "spill" for correct tense, or "while" -> "with" to remove the tense.

Reply:
Went "while" -> "with"